eng/cake/dotnet.cake

```diff
@@ -152,7 +152,6 @@ Task("dotnet-test")
             "**/Core.UnitTests.csproj",
             "**/Essentials.UnitTests.csproj",
             "**/Resizetizer.UnitTests.csproj",
-            "**/Controls.Sample.Tests.csproj"
```

Comment:
Moved these tests into `Controls.Core.UnitTests.csproj` since that is all it really was.